Fix/cross db macros (#107)
snowplow/dbt-snowplow-utils: README.md

## get_string_agg ([source](macros/utils/cross_db/get_string_agg.sql))

> This macro takes care of harmonising cross-db `list_agg`, `string_agg` type functions. These are aggregate functions that take all expressions from rows and concatenates them into a single string.

Reviewer: ```suggestion
This macro takes care of harmonising cross-db `list_agg`, `string_agg` type functions. These are aggregate functions that take all expressions from rows and concatenate them into a single string.
```

A base column and its prefix have to be provided, the separator is optional (default is ',').

> By default ordering is defined by sorting the base column ascending, if you wish to order on different column, the `order_by_column` and `order_by_column_prefix` have to be provided. If you wish to use descending order then set `order_desc` to `true`.

Reviewer: ```suggestion
By default ordering is defined by sorting the base column in ascending order. If you wish to order on a different column, the `order_by_column` and `order_by_column_prefix` have to be provided. If you wish to order in descending order, then set `order_desc` to `true`.
```

> In case the field used for sorting happens to be of numeric value (regardless whether it is stored as a string or as a real numeric value) the sort_numeric parameter should be set to true, which takes care of conversions from sting to numeric, if needed.

Reviewer: ```suggestion
In case the field used for sorting happens to be of numeric value (regardless of whether it is stored as a string or as a numeric type) the `sort_numeric` parameter should be set to true, which takes care of conversions from sting to numeric if needed.
```
